### DISCUSSION TITLE
G7 calibration logs are confusing

### DISCUSSION THREAD

**Discussion by Navid200** [NightscoutFoundation/xDrip#3644] [Q&A]
A post on facebook about G7 calibration with xDrip includes this screenshot.
![456976494_8064313687021924_1436055729562293458_n](https://github.com/user-attachments/assets/dd2c589e-5034-4b04-9a16-2b06a5b03377)
  
There is no log stating that calibration was accepted by the transmitter as there is with G6.  

I am going to open a PR to address the other two logs as they are clearly related to raw values and we know G7 does not send raw  values.  So, I will add a conditional not to print those two for G7 or regular G6.   

But, can you give me tips about the missing acknowledgement and instead having an unknown message?

**Comment by Gonzmdv** [comment]
Hello, I started with G7 nine days ago. The first day, at some point the G7 readings were  40 mg/dl higher than my finger test (checked twice), so I tried to calibrate the sensor with my Tandem Slim X2 pump. Glucose readings didn't change, so i tried to calibrate it again with the pump and later with xDrip+, several times. No change. I was so tired that finally felt asleep I when I got up the sensor was showing the same values than my blood test, so I haven't tried to calibrate it again.
When I was using the G6, the sensor was showing the new value as soon as I typed the new calibration.

**Reply by Navid200** [reply]
If you calibrated with t:slim and saw unexpected behavior, it cannot have anything to do with xDrip.
xDrip is open source.  T:slim is not.  You can contact Dexcom and ask for advice about how you need to calibrate with T:slim.

A comparison between G6 and G7 has nothing to do with xDrip either.
